tests/range_vue/example.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <p>Templates are not formatted yet ...
+    </p>
+</template>
+
+<<<PRETTIER_RANGE_START>>><script>
+let Prettier        = format => { your.js('though') }
+</script><<<PRETTIER_RANGE_END>>>
+
+<style>
+.and { css: too! important }
```

Comment:
Looks invalid css

Reply:
I copied this from the playground sample...